packages/remix-node/fetch.ts

```diff
@@ -111,29 +47,3 @@ class NodeRequest extends BaseNodeRequest {
 }
 
 export { NodeRequest as Request, NodeRequestInit as RequestInit };
-
-/**
- * A `fetch` function for node that matches the web Fetch API. Based on
- * `node-fetch`.
- *
- * @see https://github.com/node-fetch/node-fetch
- * @see https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
- */
-export function fetch(
-  input: RequestInfo,
-  init?: RequestInit
-): Promise<Response> {
-  init = { compress: false, ...init };
-
-  if (init?.body instanceof NodeFormData) {
-    init = {
-      ...init,
-      body: formDataToStream(init.body),
-    };
-  }
-
-  // Default to { compress: false } so responses can be proxied through more
```

Comment:
TODO: Double check that web-std/fetch doesn't decompress the bytes in a response so you can return a fetch() for an encoded resource directly from your loader().